apps/modernization-ui/src/apps/page-builder/page/management/header/tabs/PageTabs.tsx

```diff
@@ -1,22 +1,88 @@
-import { KeyboardEvent as ReactKeyboardEvent } from 'react';
+import { ReactNode, useRef, useState, KeyboardEvent as ReactKeyboardEvent, useEffect } from 'react';
 import classNames from 'classnames';
 import { PagesTab } from 'apps/page-builder/generated';
-
+import { Button, Icon, ModalRef, ModalToggleButton } from '@trussworks/react-uswds';
+import { AlertBanner } from 'apps/page-builder/components/AlertBanner/AlertBanner';
+import { ModalComponent } from 'components/ModalComponent/ModalComponent';
+import { AddEditTab } from 'apps/page-builder/page/management/edit/tabs/AddEditTab/AddEditTab';
 import styles from './page-tabs.module.scss';
 import { usePageManagement } from '../../usePageManagement';
+import { addTab } from 'apps/page-builder/services/tabsAPI';
+
+type AlertMessage = { type: 'warning' | 'success'; message: string | ReactNode; expiration: number };
 
 type Props = {
+    pageId: number;
     tabs: PagesTab[];
+    onAddSuccess?: () => void;
 };
-export const PageTabs = ({ tabs }: Props) => {
+
+type TabEntry = { name: string | undefined; visible: boolean; order: number };
+
+export const PageTabs = ({ pageId, tabs, onAddSuccess }: Props) => {
```

Comment:
Issue: The `PageTabs` component is used in the `PageHeader`, this is included in both the `Edit` and `Preview` pages. Management features will not be available in the Preview mode.

I don't like adding the manage tabs functionality to this component and would rather have a separate `ManageTab` component that renders just the `Manage tabs` text and handles all of the required logic. That way the `PageTabs` can still be a simple list renderer with onClick handlers.

This new component can then be added to the `PageHeader` and conditionally rendered if editing is enabled. May have to have an optional prop on the `PageHeader` to allow enabling the editing.